File: JSBASIC/121_is_prime.js
// 121	121_is_prime.js	Check whether a given number is prime or not.

let num = 29;
let isPrime = true;
if (num < 2) {
    isPrime = false;
} else {
    for (let i = 2; i <= Math.sqrt(num); i++) {
        if (num % i === 0) { isPrime = false; break; }
    }
}
console.log("Number:", num);
console.log(num, isPrime ? "is a Prime number." : "is NOT a Prime number.");

File: JavaBasic/121_is_prime.java
// 121	121_is_prime.java	Check whether a given number is prime or not.
public class _121_is_prime {
    public static void main(String[] args) {
        int num = 29;
        boolean isPrime = num >= 2;
        for (int i = 2; i <= Math.sqrt(num); i++) { if(num%i==0){ isPrime=false; break; } }
        System.out.println("Number: " + num);
        System.out.println(num + (isPrime ? " is Prime." : " is NOT Prime."));
    }
}
